samples/2048.cr

```diff
@@ -2,31 +2,40 @@
 
 require "colorize"
 
+enum Action
+  Up
+  Down
+  Left
+  Right
+  Escape
+  Unknown
+end
+
 module Screen
   TILES = {
-        0 => {:white, nil},
-        2 => {:black, :white},
-        4 => {:blue, :white},
-        8 => {:black, :yellow},
-       16 => {:white, :red},
-       32 => {:black, :red},
-       64 => {:white, :magenta},
-      128 => {:red, :yellow},
-      256 => {:magenta, :yellow},
-      512 => {:white, :yellow},
-     1024 => {:white, :yellow},
-     2048 => {:white, :yellow},
-     4096 => {:white, :black},
-     8192 => {:white, :black},
-    16384 => {:white, :black},
-    32768 => {:white, :black},
-    65536 => {:white, :black},
+        0 => {Colorize::ColorANSI::White, nil},
```

Comment:
Optional: have a private def taking two `Colorize::ColorANSI` instances. Then you can do something like:

```crystal
0 => colors(:white, :black),
```

That said, I don't mind the verbosity.